pkg/controllers/user/monitoring/clusterHandler.go

```diff
@@ -237,6 +238,9 @@ func (ch *clusterHandler) getExporterEndpoint() (map[string][]string, error) {
 	controlplaneLabels := labels.Set{
 		"node-role.kubernetes.io/controlplane": "true",
 	}
+	windowsNodeLabels := labels.Set{
+		"kubernetes.io/os": "windows",
```

Comment:
Are we supposed to only support k8s 1.15 for windows cluster? If not, we should consider using `beta.kubernetes.io/os=windows` also.

Reply:
this label ```kubernetes.io/os``` release in k8s 1.14, we also support windows begin 1.14, so don't need beta